apis/duck/v1beta1/addressable_types.go

```diff
@@ -24,8 +24,8 @@ import (
 	"k8s.io/apimachinery/pkg/runtime"
 
 	"knative.dev/pkg/apis"
-	"knative.dev/pkg/apis/duck"
-	v1 "knative.dev/pkg/apis/duck/v1"
+	"knative.dev/pkg/apis/duck/ducktypes"
+	"knative.dev/pkg/apis/duck/v1"
```

Comment:
<!--gotool-bot-->
Format Go code:
```suggestion

```

Reply:
This would break compilation

Reply:
why would it?

Reply:
Because it removes an import that is being referenced?

Reply:
I guess we need to call it `duckv1` to conform to our repository conventions.
I think at some point it saw both named and unnamed import.

Reply:
there is already a `duckv1`. My PR does not change anything about the existing imports other than removing the useless `v1` prefix.